larg4/pluginActions/ParticleListAction_service.cc

```diff
@@ -210,6 +206,26 @@ namespace larg4 {
     }
   }
 
+  //-----------------------------------------------------------------
+  // read up the target ID map to return what the stored track ID 
+  // should be for a given trackid
+  //
+  /*
+  int
+  ParticleListActionService::getStorableTrackID(int trackid) const
+  {
+    auto itr = fTargetIDMap.find(trackid);
+    if(itr == fTargetIDMap.end()){
+      mf::LogWarning("ParticleListActionService::getStorableTrackID")
+	<< "Track ID " << trackid << " not found in the TargetIDMap.  "
+	<< "Returning original trackid, but there may be issues with truth-matching.";
+      return trackid;
+    }
+    return itr->second;
+  }
+  */
```

Comment:
Suggest removing this commented-out code.

Reply:
done